manifest.yml

```diff
@@ -6,6 +6,7 @@ memory: 1024M
 health-check-type: process
 # health-check-http-endpoint: /robots.txt
 stack: cflinuxfs3
+services:
```

Comment:
Does this still successfully deploy with both `cf push` and `cf bgd` if no services are added?

Reply:
no, i've changed it so this is written in along with the the name when present.